src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -9,6 +9,9 @@
 using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
 using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient;
 using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.Transmission;
+#if !NET
```

Comment:
Consider `!NET && !NETSTANDARD2_1`.

This PR is for fixing .NET Framework consumers. netstandard2.1 libraries cannot be referenced by .NET Framework.